events/reporter.go

```diff
@@ -203,17 +203,15 @@ func ReportError(err NodeError) {
 	defer mu.RUnlock()
 
 	if reporter != nil {
-		if reporter.blocking {
-			reporter.channelError <- err
-			log.Debug("reported error: %v", err)
-		} else {
+		log.Info("about to report error updates for error : %v", err)
```

Comment:
please use the new log format:

```suggestion
		log.With().Info("about to report error updates for error", log.Err(err))
```